users/common/strings.py

```diff
@@ -45,13 +45,13 @@ class CustomUserModelText:
 
 
 class ValidationErrorText:
-    VALIDATION_ERROR_EMAIL_MESSAGE = "The given email must be set"
+    VALIDATION_ERROR_EMAIL_MESSAGE = "Enter your email address, please."
```

Comment:
This sounds for me like a begging for correct email address :D Maybe ```Please enter correct email address``` ?

Reply:
Done:)